iceoryx_posh/include/iceoryx_posh/internal/popo/base_server.hpp

```diff
@@ -136,15 +137,15 @@ class BaseServer
     /// @brief port
     /// @return const accessor of the underlying port
     ///
-    const Port& port() const noexcept;
+    const PortT& port() const noexcept;
```

Comment:
Again not a must have but why not use `PortType` when you have it - reads better.